src/Makevars

```diff
@@ -0,0 +1,4 @@
+## We want C++11 
+CXX_STD = CXX11
+PKG_CXXFLAGS = -I../inst/include
+PKG_LIBS = -lstdc++ -ldl
```

Comment:
I don't think we want/need those for `PKG_LIBS`. They should be added automatically.

Reply:
Done in 18bc559

Reply:
Excellent.